lib/dfid-transition/transform/unescape_html.rb

```diff
@@ -0,0 +1,12 @@
+require 'cgi'
+
+module DfidTransition
+  module Transform
+    module UnescapeHtml
```

Comment:
This seems pretty specific for a module (there'd only ever be one function in it). How about `module Html...def self.unescape`?